XML3-Homework/Gardiner_XML3.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xml>
+    <meta source="https://www.depaulatrading.com/blogs/news/icarus"
+        title="The Story of Icarus from Greek Mythology"/>
+
+    <section title="Good Old Dad">
```

Comment:
Hey, I see you're using spaces in your attribute values. got a good reason for that? @jbg5721